File: WebApplication1/QueryHandler/Query/TvSeriesQuery.cs
using MediatR;
using WebApplication1.DTO.Response;
using WebApplication1.Models;

namespace WebApplication1.QueryHandler.Query
{
    public class TvSeriesQuery : IRequest<List<TvSeriesResponse>>
    {
        public string? title { get; set; }
        public string? genreName { get; set; }
        public DateTime dateTime { get; set; }
        public int? seasons { get; set; }
        public int? episodes { get; set; }
        public string? network { get; set; }
        public EStatus? status { get; set; }
        public int? isAvg { get; set; }

        public string? sortByField { get; set; }
        public bool isDescending { get; set; }
    }
}

File: angular-movieFrontend/src/ClientPages/MainPages/tv-series-web/TvSeriesQuery.ts
export interface TvSeriesQuery{
    TitleSearch: string | null;
    MinRating: number | null;
    ReleaseYear: number | null;
    genreName: string | null;
    status: string | null;
    
    SortByField: string | null;
    IsDescending: boolean;
}